src/Microsoft.ML.AutoML/Sweepers/SweeperProbabilityUtils.cs

```diff
@@ -115,6 +118,10 @@ public static float[] ParameterSetAsFloatArray(IValueGenerator[] sweepParams, Pa
             return result.ToArray();
         }
 
+        private static T? GetIfIParameterValueOfT<T>(IParameterValue parameterValue)
+            where T : struct =>
+            parameterValue is IParameterValue<T> pvt ? pvt.Value : default(T?);
```

Comment:
Instead of "default(T?)" can't we just have "null"? I think it would be more readable if it had explicitly return null.

Reply:
On the other hand, I guess it's fine to keep it like this anyway.